extra-web/telegram-desktop/spec

```diff
@@ -1,5 +1,7 @@
-VER=2.4.4
+VER=2.4.6
 SRCS="tbl::https://github.com/telegramdesktop/tdesktop/releases/download/v$VER/tdesktop-$VER-full.tar.gz \
-      git::rename=tg_owt;commit=c73a4718cbff7048373a63db32068482e5fd11ef::https://github.com/desktop-app/tg_owt"
-CHKSUMS="sha256::88799708daedaf5c132752259945b4e1b5dcbb62385661e2cca7b8e5d58c9026 SKIP"
+      git::rename=tg_owt;commit=e8fcae73947445db3d418fb7c20b964b59e14706::https://github.com/desktop-app/tg_owt"
+CHKSUMS="sha256::2c6f97ca079c3f9aad700f133d8bcd45ae4cfa32423e91d02c79a163664c13a4 SKIP"
 SUBDIR=tdesktop-$VER-full
+
+# The update of tg_owt follows the latest commit of the upstream master
```

Comment:
Move this directly after `VER=`, above the `SRCS=` values, and please reword it as follows:

"Update tg_owt to the latest Git snapshot when updating Telegram Desktop."